Autopilot/Inc/ultrasonic.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+
+Ultrasonic Sensor driver (HC-SR04) 
+By: Dhruv Upadhyay
+September 2021
+
+*/
+
+
+#ifndef ULTRASONIC_HPP
+#define ULTRASONIC_HPP
+
+#include "stdint.h"
+#include "stdbool.h"
+#include "stm32f765xx.h"
+#include "stm32f7xx_hal_tim.h"
```

Comment:
Let's include this in the .cpp file so any file that includes `ultrasonic.hpp` is still unit testable